Content.Shared/Access/Systems/AccessReaderSystem.cs

```diff
@@ -67,8 +68,9 @@ private void OnInit(EntityUid uid, AccessReaderComponent reader, ComponentInit a
 
         private void OnEmagged(EntityUid uid, AccessReaderComponent reader, ref GotEmaggedEvent args)
         {
-            // no fancy conditions
             args.Handled = true;
+            reader.Enabled = true;
```

Comment:
shouldn't this be false if emagging something with access reader always lets people through